docs/source/configuration/configuration-gconstruction.rst

```diff
@@ -101,10 +105,11 @@ An example
 ............
 Below shows an example that contains one node type and an edge type. For a real example, please refer to the :ref:`input JSON file <input-config>` used in the :ref:`Use Your Own Graphs Tutorial <use-own-data>`.
 
-.. code-block:: yaml
+.. code-block:: json
 
     {
-        nodes: [
+        "version": "gconstruct-0.1",
```

Comment:
should we use something like `v0.1` as the version number?

Reply:
Sure, I can make this change.